### DISCUSSION TITLE
How intentional prompts create game-changing suggestions 🤩   [AI2575D] [Day 1 Universe]

### DISCUSSION THREAD

**Discussion by blackgirlbytes** [community/community#74629] [Copilot Conversations]
## How intentional prompts create game-changing suggestions 🤩
**Speaker:** Rizel Scarlett, Staff Developer Advocate at GitHub
**Time:** Wednesday, Nov 8, 2:00 PM - 2:25 PM PT
**Social:** You can follow me on.. [Twitter](https://twitter.com/blackgirlbytes) | [Linkedin](https://www.linkedin.com/in/rizel-bobb-semple/) | [Instagram](https://www.instagram.com/blackgirlbytes) | [Mastodon](https://hachyderm.io/@blackgirlbytes)

> Generative AI coding assistants are revolutionizing the way we code, but it can feel frustrating when these tools produce irrelevant or absent suggestions. Just like with any emerging tech, there's a learning curve to using the new tools. Join GitHub Copilot–enthusiast Rizel Scarlett as she shares proven strategies and expert tips to transform your random prompts into intentional ones, unlocking a world of game-changing suggestions. Get inspired, get creative, and get ready to embrace the future of coding!

Thank you for joining my/our session/workshop! If you have any questions or feedback regarding
what was presented, please feel free to add them in the comments and I'll do my best to
answer.

**Comment by ghostinhershell** [comment]
Really excited about checking this out @blackgirlbytes! 🚀

**Reply by blackgirlbytes** [reply]
aw thank you! Were you there in person. I didn't catch you!
